packages/ecommerce-app-base/src/SkuPicker/SkuPicker.tsx

```diff
@@ -22,6 +22,7 @@ export interface Props {
   skuType?: string;
   makeSaveBtnText?: MakeSaveBtnTextFn;
   hideSearch?: boolean;
+  showSearchBySkuOption?: boolean;
```

Comment:
ruminated on this name for a while 😄 wanted to be as explicit as possible about what it renders when true - was between `showSearchBySkuOption` or `showSearchBySkuFilter` ? Any other thoughts here are welcome!

Reply:
I like how you renamed it, thank you for making it more explicit!